components/devtools/Cargo.toml

```diff
@@ -11,6 +11,7 @@ name = "devtools"
 path = "lib.rs"
 
 [dependencies]
+chrono = "0.4"
```

Comment:
Is it possible to use `std::time` instead of introducing a new dependency in this crate on the chrono crate?

Reply:
I tried, the only missing function is `to_rfc3339 `, can not found it in time and it exists in chrono

Reply:
Okay. Would it be possible to use `chrono` only for this call to `to_rfc3339`? That way we limit our use of third-party libraries to the minimum and can replace or standardize in the future more easily.

Reply:
I would only keep `chrono` for `components/devtools/actors/network_event.rs` which is where you need `to_rfc3339` and use `std::time` structs in all other places. I would use something like `Instant` all over the place and only convert to a `chrono` type right before you need the string version of the timestamp.

Reply:
Thanks for suggestion, I used it.